fee_allocator/allocations/output_for_msig/2023-12-21_aura_poc.csv

```diff
@@ -0,0 +1,106 @@
+target,platform,amount
```

Comment:
shouldn't we be writingb to the main file in the main branch/not poc.  Also can't find teh code anywhere here that changes the filename, but maybe I'm bliind.